src/figaro-validation-service/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/applicant/ApplicantController.java

```diff
@@ -92,6 +92,8 @@ public ValidateApplicantPartyIdResponse validateApplicantPartyId(
     @GetMapping(value = "/locateMatchingApplicants", produces = { MediaType.APPLICATION_XML_VALUE })
     @ApiOperation(value = "locateMatchingApplicants", response = LocateMatchingApplicantsResponse.class, tags={ APPLICANT_API })
     @ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = LocateMatchingApplicantsResponse.class) })
+    @SuppressWarnings("java:S107")
```

Comment:
any suppress we're doing needs a comment imo

Reply:
prolly should've mentioned this earlier, brought it up with @alexjoybc yesterday. apologies if I'm springing this a bit late.

Reply:
The line below it had the comment